flocker/node/test/test_loop.py

```diff
@@ -538,8 +539,8 @@ def test_convergence_sent_state_fail_resends_alternating(self):
             client=client, configuration=configuration, state=state))
 
         # Wait for all three iterations to occur.
-        reactor.advance(_UNCONVERGED_DELAY.delay_seconds)
-        reactor.advance(_UNCONVERGED_DELAY.delay_seconds)
+        reactor.advance(_UNCONVERGED_DELAY)
+        reactor.advance(_UNCONVERGED_DELAY*2)
```

Comment:
If I'm reading this test correctly, the reactor should now advance by the value provided by `UnconvergedDelay.sleep()` which in this case would be `_UNCONVERGED_DELAY * 4`. It would probably be better to have the factor that the delay increases by as a constant.